app/src/main/kotlin/com/wire/android/ui/common/topappbar/CommonTopAppBar.kt

```diff
@@ -100,7 +99,7 @@ private fun ongoingCallLabel(
 @Composable
 private fun clearStatusBarColor() {
     val backgroundColor = MaterialTheme.wireColorScheme.background
-    val darkIcons = MaterialTheme.wireColorScheme.useDarkSystemBarIcons
+    val darkIcons = MaterialTheme.wireColorScheme.ongoingCallLabelShouldUseDarkIcons
```

Comment:
If you clear status bar color to the default one then you also need to use the default icons flag `useDarkSystemBarIcons` here 🙂 

That's because you set status background color to be `MaterialTheme.wireColorScheme.background` so you need to use corresponding icons flag `useDarkSystemBarIcons`

Reply:
true! missed that again